roles/openshift_health_checker/library/aos_version.py

```diff
@@ -32,11 +32,14 @@
     PKG_MGR = "yum"
 except ImportError as err:
     YUM_IMPORT_EXCEPTION = err
-try:
-    import dnf  # pylint: disable=import-error
-    PKG_MGR = "dnf"
-except ImportError as err:
-    DNF_IMPORT_EXCEPTION = err
+
+# Import and use dnf only if yum is missing
```

Comment:
Hmm, this could go for now, but long term we might want to do the same as the `package` action plugin that dispatches to the `yum` or `dnf` module, and use the `ansible_pkg_mgr` fact:

https://github.com/ansible/ansible/blob/1b8f4558f76fea0d1533ca6581815429c6f92ee8/lib/ansible/plugins/action/package.py#L55

Since modules don't have access to any of Ansible's internals, we'd pass in as an arg either `yum` or `dnf` from the check in `package_availability.py`.

Reply:
I did not mean to leave the conditional in. Should try to import both regardless, and use the one indicated by `ansible_pkg_mgr`.